src/internal/connector/onedrive/restore.go

```diff
@@ -85,10 +85,10 @@ func RestoreCollections(
 		nil
 }
 
-// restoreCollection handles restoration of an individual collection.
-// @returns Integer representing totalItems, restoredItems, and the
-// amount of bytes restored. The bool represents whether the context was cancelled
-func restoreCollection(
+// RestoreCollection handles restoration of an individual collection.
+// returns the metrics gathered during restoration.
+// returns true if the context was cancelled.
```

Comment:
This is a little ambiguous with two statements that say `returns`. If the previous was overkill, there should be a healthy middle ground that gives clarity to the function.

Reply:
Fair, I've updated it.